### PR TITLE
Setup objectbox

### PR DESCRIPTION
**Pull request by kordianbruck**
WIP #443

### CODE REVIEW

File: app/src/main/java/de/tum/in/tumcampusapp/entities/MyBoxStore.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package de.tum.in.tumcampusapp.entities;
+
+
+import android.content.Context;
+
+import io.objectbox.BoxStore;
+
+public class MyBoxStore {
```
Comment (pfent) [inline app/src/main/java/de/tum/in/tumcampusapp/entities/MyBoxStore.java]: MyBoxStore is probably not a good name…

**Comment by pfent** [comment]
Ehhh…
> Once a an updated License OR a final release is available, you need to **stop using any of the material** made available to you with this beta license and switch to the then current release version with its terms and conditions  

http://greenrobot.org/objectbox/license/
Does not seem like this library will be open-source or under a compatible license. Probably can't merge it with such a license…

**Comment by kordianbruck** [comment]
I wouldn't worry about the license for now. I mean we can ask in the public repository, what happens when the stable comes out. Good that you spotted that problem.

File: app/src/main/java/de/tum/in/tumcampusapp/entities/MyBoxStore.java (hunk 1 above)
Comment (kordianbruck) [inline app/src/main/java/de/tum/in/tumcampusapp/entities/MyBoxStore.java]: Kay, renamed.

**Comment by kordianbruck** [comment]
@pfent their answer to the question about licensing https://github.com/greenrobot/ObjectBox/issues/29#issuecomment-276294643

**Comment by kordianbruck** [comment]
Ping

**Comment by kordianbruck** [comment]
On hold for some time.

**Pull request closed without merging**